test/utils/integration/integration.go

```diff
@@ -60,7 +60,7 @@ const (
 	namespacePrefixLength    = 5
 	TwoMinTimeOut            = 120 * time.Second
 	BigPollingInterval       = 3 * time.Second
-	BigTimeOut               = 30 * time.Second
+	BigTimeOut               = 60 * time.Second
```

Comment:
do you you really want to have 1 minute here. Maybe 40 seconds?

Reply:
I have tested it 8 times with 1 minute and its seems stable, so lets keep this 1 minute.